### DISCUSSION TITLE
QR Presets / multiple

### DISCUSSION THREAD

**Discussion by WURS5000** [gopro/labs#299] [Feature Requests]
Hey there, 

since we can't set different lab's settings on various presets, i was think about if we could get something like a "master-qr" code.

So my suggestion would be, instead of sending one command per QR, we can send multiple commands, including !RESET in front:

_Example_
`!RESET;!MPRXY=1;!MEXPX=120;!MLOGB=400;!MBITR=190;!MNR01=1;!MWIDE=1;and so on;and so on`

Somehow the max-shutter QR allready kinda manage to do this, by setting max shutter angle AND max iso with one QR.

Thanks and have a good weekend

**Comment by dnewman-gpsw** [comment]
Huh?!?  

This is works now:  !MPRXY=1!MEXPX=120!MLOGB=400!MBITR=190!MNR01=1!MWIDE=1  as a single QR code.  You never need to separate them.

**Reply by WURS5000** [reply]
Ouh, thank you - very helpful, didn't knowed that.
